prophesee_ros_driver/src/prophesee_ros_publisher.cpp

```diff
@@ -141,22 +191,52 @@ void PropheseeWrapperPublisher::publishCDEvents() {
                     return;
 
                 if (ev_begin < ev_end) {
-                    // Define the message for a buffer of CD events
-                    prophesee_event_msgs::EventArray event_buffer_msg;
+                    // Compute the current local buffer size with new CD events
                     const unsigned int buffer_size = ev_end - ev_begin;
-                    event_buffer_msg.events.resize(buffer_size);
 
-                    // Header Timestamp of the message
-                    event_buffer_msg.header.stamp.fromNSec(start_timestamp_.toNSec() + (ev_begin->t * 1000.00));
+                    // Get the current time
+                    event_buffer_current_time_.fromNSec(start_timestamp_.toNSec() + (ev_begin->t * 1000.00));
```

Comment:
why do you need it here ?
You are doing it 2 times : lines 198 and 218 

Could you remove one of these lines ?

Reply:
I don't understand the comment.
It is different. One is the first event, the second one is the last event of the buffer.

Reply:
sorry, didn't see it